coriolis/osmorphing/manager.py

```diff
@@ -13,6 +13,13 @@
 from coriolis.osmorphing.osmount import factory as osmount_factory
 from coriolis.osmorphing.osdetect import manager as osdetect_manager
 
+
+opts = [
+    cfg.IntOpt('default_osmorphing_operation_timeouts',
```

Comment:
NIT: `timeouts` should be singular. If anything should be plural it's `operations`, but the general `operation` should be fine IMO.